### DISCUSSION TITLE
In-class thinking cap!

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#58] [Thinking Caps]
Reflect on what the process of communicating and what that added to your experience:

•As a mentee what was your mentor able to add?

•As a mentor, what new things have you learned from this short exchange from the mentee that is related to your project?

**Comment by nesslee123** [comment]
My mentor was able to help me think about things that I may have missed which helped me find new ways to attempt to solve my problem.

It helped me realize that asking questions also helps others look at things with a new perspective.

**Comment by kchhoeu** [comment]
The process of communicating added to the experience, that it made me realize a lot of us are in the same page, and that it is okay to have stumps in the road. As a mentee the mentor added pretty valuable insight. I think it was nice to talk about the struggles but also just have conversations with new people

**Comment by giaari15** [comment]
My mentor was able to talk about their own experience with a similar issue and was able to guide me to find solutions. I learned a bit more about how transparent images show up. It was good to be able to talk through an issue with different perspective.

**Comment by chrissanrow** [comment]
My peer mentor affirmed by ideas and encouraged me to try what I thought might be difficult. As a mentor, while my mentee didn't really have any problems, I found it enlightening to inquire about my mentee's process in working on their final project.

**Comment by pwinnie2026** [comment]
My mentor was able to empathize with what I was saying and allowed me to look at my experience from a different viewpoint. As a mentee, I also learned more about how different communication styles could work from someone whose group already completed their rough draft and related this to my project. I really enjoyed this experience, and it was good to relate to different topics and learn that we had more things in common than I thought.

**Comment by NellieNguyen149** [comment]
As a peer mentee, my mentor helped me understand the bigger picture because at a certain point in my project, I think I got too "close" the certain details that I didn't step back to fully view everything. I also got to learn more about different ways to approach my project/trying other ways of thinking to help move forward!

As a peer mentor, I learned a lot from listening to my mentee, and to be honest, I felt a little under-qualified to suggest or add anything. However, I realized my role as a mentor wasn't expected to give concrete/solid advice on how to move forward, but instead to serve as someone to help the person hear their own thoughts and run it through a new perspective. :D

**Comment by stinest** [comment]
I was able to have an extremely friendly and comforting conversation with my mentor who let me see the solutions to my problems, also sharing with me her own experiences to help me better understand other perspectives!

**Comment by zhamic7** [comment]
My mentor helped me think of a possible way to format site popups better if they're getting too long! As a mentor, I realized a lot of us are facing the same types of coding challenges (i.e., CSS ...).

**Comment by sjeyakum** [comment]
My mentor asked follow-up questions that helped me realize that the issue I brought up wasn't really the main issue. While I was talking about one issue (legend markers), I realized through our conversation that the issue I was concerned about was the organization of the legend content.  

As a mentor, I realized that we shared similar problems. I definitely had a tough time asking follow-up questions, but we were able to talk a lot so hopefully that helped to some extent (definitely want to improve on asking effective follow-up questions though).

**Comment by Keiya-Wada** [comment]
My mentor helped me by relating to the problems I had, which eased me mentally, that I wasn't the only one having these problems. As a mentee, actively thinking of questions to ask allowed me to gain a better understanding on my partner's problems and project. Overall, I was able to gain new perspectives that were much needed in my project.

**Comment by kianaptran** [comment]
As a mentee, I could share my struggles with coding with my mentor, who was also someone new to coding! It was refreshing to relate to someone who was also just learning about the material, and we mutually understood each other's struggles. My mentor told me some skills and methods she used when she got stuck on any lab/project, which I will adopt moving forward. Some of these methods involve styling code she learned online.

As a mentor, I learned about some communication issues other groups face. Especially when it comes to dividing up the work and having a mutual understanding of who is doing what task; it can be easy to have one person do the job while someone is unaware. Relating this to my project, I could reflect on the communication within my group; fortunately, we have had good communication throughout the project.

**Comment by jasminereyz** [comment]
As a mentee, I was able to gain different insight on what could make our project more concrete as this is something my partner and I have struggled with the most. Despite being very passionate about our topic, this is also what my partner and I struggle with the most in making sure we stay true to our specific goals. So, talking to someone who isn't my partner or Albert really helped a lot and I feel much more prepared for our final project!

As a mentor, I  realized that I should not be as stressed or hard on myself as many other of my classmates are exactly where my partner and I are. I feel that talking in this breakout room helped both of us understand that as long as we stay true to the guidelines and to empower the voices of our organization, the rest will fall into place
